### PR TITLE
add test cases for input, enumerate all states

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

add test cases for input, enumerate all states

### CODE REVIEW

File: src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap [hunk 1]
```diff
@@ -1,6 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Input Editable white input field matches the snapshot 1`] = `
+exports[`Input Input field when mandatory and no label matches the snapshot 1`] = `
```
Comment (akroon3r) [inline src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap]: double input

Comment (akroon3r) [reply src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap]: not sure if meant or not

File: src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+    type="text"
+  />
+  <br />
+  <span
+    className="error"
+  />
+</div>
+`;
+
+exports[`Input Input field when mandatory and with label matches the snapshot 1`] = `
```
Comment (akroon3r) [inline src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap]: same, so perhaps it was meant.

File: src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap]: yup fixed

File: src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap (hunk 2 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap]: nope, it is fixed now - i fixed the wording

**Comment by akroon3r** [comment]
Looks like you addressed my feedback before I submitted. 🚢

**Pull request merged**